File: 3379-ScoreOfAString/3379-ScoreOfAString.py
# Last updated: 2/22/2026, 1:54:35 PM
class Solution:
    def scoreOfString(self, s: str) -> int:
        score = 0
        for i in range(1,len(s)):
            score+=abs(ord(s[i]) - ord(s[i-1]))
        return score
        

File: 3379-ScoreOfAString/3379-ScoreOfAString.c
// Last updated: 2/22/2026, 1:54:40 PM
int scoreOfString(char* s) {
    int score = 0;
    for(int i = 0 ; *(s+1)!='\0' ; i++)
    {
        score+=abs(*s - *(s+1));
        s++;
    }
    return score;
}